docs/contributing/CHANGELOG_TEMPLATE.md

```diff
@@ -4,26 +4,21 @@
 
 💥 Breaking changes:
 
-- Pull Request Title goes here
+Heading (h4) to summarise change goes here
```

Comment:
@EoinShaughnessy Can we mark this up as an actual h4 heading, e.g:

```suggestion
#### Heading (h4) to summarise change goes here
```

Reply:
Sounds good!